pandas/core/arrays/base.py

```diff
@@ -519,6 +519,86 @@ def argsort(
         result = nargsort(self, kind=kind, ascending=ascending, na_position="last")
         return result
 
+    def argmin(self):
+        """
+        Return the minimum argument indexer.
+
+        Returns
+        -------
+        scalar
+            Minimum argument indexer.
+
+        See Also
+        --------
+        ExtensionArray.argmax
+        """
+        if len(self) == 0:
+            raise ValueError("attempt to get argmin of an empty sequence")
+
+        return self.argsort()[0]
+
+    def min(self):
+        """
+        Return the minimum value.
+
+        Returns
+        -------
+        scalar
+            Minimum value.
+
+        See Also
+        --------
+        ExtensionArray.max : Return the maximum value of the object.
+        Series.min : Return the minimum value in a Series.
+        DataFrame.min : Return the minimum values in a DataFrame.
+        """
+        if len(self) == 0:
```

Comment:
hmm, we do allow this for Series (we return nan/NA). This is more like how numpy does it, so ok. cc @jorisvandenbossche @TomAugspurger

Reply:
@jorisvandenbossche @TomAugspurger
What do you think? I will go ahead with jreback's approach. Thanks.

Reply:
Mathcing NumPy makes sense I think.

Reply:
@jorisvandenbossche Would you tell us your thought? Thanks.